docs/sql/general/lexical-structure.rst

```diff
@@ -28,6 +28,26 @@ String literals are defined as an arbitrary sequence of characters that are
 delimited with single quotes ``'`` as defined in ANSI SQL, for example
 ``'This is a string'``.
 
+In addition, CrateDB supports dollar quoted strings to help avoid escaping
+single quotes within single quoted strings.
+For example, ``'I''m a string'`` can be re-written as
+``$TAG$I'm a string$TAG$``, where the matching pair of ``TAG`` can be zero or
```

Comment:
I don't get this phrase: `can be zero or more characters`?

Reply:
Thanks, I appended `in length`, hope it is clearer.